site-src/guides/serve-multiple-genai-models.md

```diff
@@ -180,9 +180,16 @@ kubectl get httproute llm-phi4-route -o yaml
           -H "Content-Type: application/json" \
           -d '{
                 "model": "meta-llama/Llama-3.1-8B-Instruct",
-                "prompt": "Linux is said to be an open source kernel because ",
-                "max_tokens": 100,
-                "temperature": 0
+                "messages": [
+                  {
+                    "role": "developer",
+                    "content": "You are a helpful assistant."
+                  },
+                  {
+                    "role": "user",
+                    "content": "Hello!"
```

Comment:
can we keep previous prompt?
I'm not sure if it appears in more places, so it's probably better to keep the prompt text in case it does, so it will look the same in all places:
```suggestion
                    "content": "Linux is said to be an open source kernel because "
```